sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2131,6 +2131,14 @@ object SQLConf {
       .booleanConf
       .createWithDefault(true)
 
+  val SORT_MERGE_JOIN_BATCH_SIZE =
```

Comment:
If I understand correctly, is this new configuration used to avoid OOM? I think there should add  a new case to show the original oom will not happen after this pr